maintainers/maintainer-list.nix

```diff
@@ -10485,6 +10485,12 @@
     githubId = 40720523;
     name = "MangoIV";
   };
+  manipuladordedados = {
+    email = "manipuladordedados@gmail.com";
+    github = "manipuladordedados";
+    githubId = 1189862;
+    name = "Valter Nazianzeno";
+  };
```

Comment:
This needs to be the first commit, and it should be separate commit. See this for inspiration https://github.com/NixOS/nixpkgs/pull/250939/commits.